src/sentry_transport.h

```diff
@@ -89,4 +89,10 @@ sentry_prepared_http_request_t *sentry__prepare_http_request(
  */
 void sentry__prepared_http_request_free(sentry_prepared_http_request_t *req);
 
+/**
+ * Gzip
+ */
+bool sentry_gzipped_with_compression(const char *body, const size_t body_len,
+    char **compressed_body, size_t *compressed_body_len);
+
```

Comment:
this seems to be local to the transport, so not really necessary to expose it via a header.
Also the naming convention uses two underscores (`sentry__XXX`) for internal functions.

Reply:
Yes, I has updated `sentry_gzipped_with_compression` to `sentry__gzipped_with_compression.`